swift.go

```diff
@@ -239,10 +242,10 @@ func (c *Connection) setDefaults() {
 		c.Retries = DefaultRetries
 	}
 	if c.ConnectTimeout == 0 {
-		c.ConnectTimeout = 10 * time.Second
+		c.ConnectTimeout = 60 * time.Second
```

Comment:
I think the changes from here down in this file are probably just testing changes?  If you want to propose we change the timeouts can you do it in a separate pull request please?